### ISSUE TITLE
Training from scratch issues

### ISSUE THREAD

**Issue by davidjimenezphd** [xiaoboCASIA/SV-X-Softmax#5]
I have reimplemented the SVX-loss function in tensorflow and I've made unit tests to verify the correct behaviour of it. However, when I train a Resnet50 architecture from scratch using SVX loss and MS1Mv2 dataset (the purged one) the loss diverges rapidly. Have you experienced something similar? Any recommendation or idea?

**Comment by CloudHerods** [comment]
hi, i have the same issue in mxnet!
